spec/draper/decorator_spec.rb

```diff
@@ -664,6 +664,26 @@ def hello_world
           expect{decorator.hello_world}.to raise_error NoMethodError
           expect(decorator.methods).not_to include :hello_world
         end
+
+        context 'When decorator overrides a public method defined on the object with a private' do
```

Comment:
I believe it's preferred to not capitalize these:

```suggestion
        context 'when decorator overrides a public method defined on the object with a private' do
```